pkg/hooks/loader.go

```diff
@@ -162,6 +165,18 @@ func (h *Hook) AppendMocks(m *models.Mock, ctx context.Context) error {
 	return nil
 }
 
+func (h *Hook) RemoveUnusedMocks(testSet string) error {
+	mocks, err := h.GetUsedMocks()
```

Comment:
also is it safe to directly get usedmocks without using mutex? like can there be any race condition?

Reply:
This function is only called when a test set run is completed, so at a time only control will access this function.